internal/repository/conflict_resolution_repository.go

```
@@ -0,0 +1,667 @@
+		}
+	case "ai", "last_write_wins":
+		if _, err := uuid.Parse(input.AssessmentAttemptID); err != nil {
+			return fmt.Errorf("assessment_attempt_id is required: %w", err)
+		}
+	default:
+		return errors.New("conflict resolution method is unsupported")
+	}
+	return nil
+}
+
+func sortedConflictResolutionDocuments(values []RelationshipConflictResolutionDocument) []RelationshipConflictResolutionDocument {
```

golangci-lint (2.12.2)

[error] func sortedConflictResolutionDocuments is unused

(unused)